tests/test_runs/test_run_functions.py

```diff
@@ -921,7 +986,34 @@ def test_initialize_model_from_run(self):
                 ("Estimator", GaussianNB()),
             ]
         )
-        task = openml.tasks.get_task(1198)
+        task_meta_data = {
+            "task_type": TaskType.SUPERVISED_CLASSIFICATION,
+            "dataset_id": 128,  # iris
+            "estimation_procedure_id": 1,
+            "target_name": "class",
+        }
+        _task_id = check_task_existence(**task_meta_data)
+        if _task_id is not None:
+            task_id = _task_id
+        else:
+            task_meta_data["task_type"] = TaskType.SUPERVISED_CLASSIFICATION
```

Comment:
This code now exists twice, can this be put into its own function?

Reply:
This one is still open, right?

Reply:
I think I updated it, now the metadata is defined all at [one place](https://github.com/openml/openml-python/blob/d95b5e6b836a52870a76ca79c76d9954a39d00fe/tests/test_runs/test_run_functions.py#L989), and no key is being updated separately.